medusa/providers/torrent/html/freshontv.py

```diff
@@ -58,7 +58,8 @@ def __init__(self):
         }
 
         # Proper Strings
-        self.proper_strings = ['PROPER', 'REPACK', 'REAL']
+        # Provider always returns propers and non-propers in a show search
+        self.proper_strings = ['']
```

Comment:
Yes, I would expect that?! If there are no propers configured, it shouldnt search for it?